### ISSUE TITLE
A11y_Finance Digital Assistant_Profile_like_AI4W-Ensures buttons have discernible text (.webchat__suggested-action__button[aria-keyshortcuts="Alt\ \+\ Shift\ \+\ A"])

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4333]
### Environment Details:

- Application Name: Finance Digital Assistant.
- #URL: [Finance Digital Assistant (microsoft.com)](https://staging-fda.microsoft.com/fda/?&hostappid=1ED4BBD7-CF02-41D2-8363-8F21EF7F07F5)
- OS Details:
Microsoft Windows 11 Enterprise Version 21H2(OS build: 22000.318)
Chromium Edge Version: Version 100.0.1185.44 (Official build) (64-bit)
### Issue:
Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)
### Target application: 
Finance Digital Assistant - https://staging-fda.microsoft.com/fda/?&hostappid=1ED4BBD7-CF02-41D2-8363-8F21EF7F07F5
### Element path: 
.react-film__filmstrip__item:nth-child(1) > .webchat__suggested-action.webchat--css-ohhjn-xccdat.webchat--css-ohhjn-oe4onf > .webchat__suggested-action__button[aria-keyshortcuts="Alt\ \+\ Shift\ \+\ A"]
### Snippet: 

<button aria-keyshortcuts="Alt + Shift + A" class="webchat__suggested-action__button" type="button"><img class="webchat__suggested-action__image" src="/images/thumbs-up-icon@2x.png"><span class="webchat__suggested-action__text webchat__suggested-actions__button-text"> </span></button>

### How to fix:
**Fix any of the following:**

1.   Element does not have inner text that is visible to screen readers
2.   aria-label attribute does not exist or is empty
3.   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
4.   Element has no title attribute
5.   Element's default semantics were not overridden with role="none" or role="presentation"

### MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#WCAG4.1.2;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Aug21;#Linked:Bug8884824;#LinkedBug;

**Comment by KattaBhargav** [comment]
Bug copied from Azurebot team Bug id:-[BUG 8884824](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/8884824?src=WorkItemMention&src-action=artifact_link)

**Comment by compulim** [comment]
Thanks @KattaBhargav. This will take some time, my backlog of accessibility stuff is quite full right now. 😁

**Comment by compulim** [comment]
There are two things need to be done to resolve this issue:

1. Web Chat to support `imageAltText` field
   - Currently, the schema did specify the `imageAltText` field
   - However, the schema did not specify where to use this field
   - https://github.com/microsoft/botframework-sdk/issues/6527
3. Bot author needs to use this field
   - Currently, the bot does not reply with `imageAltText`
   - ![image](https://user-images.githubusercontent.com/1622400/180842493-4d246173-1a6b-446e-bc09-a0da537d2bd5.png)

**Comment by compulim** [comment]
This is fixed on Web Chat side in #4441.

The bot author would still need to modify their bot to use this field. Instructions have been given above.
